Gruntfile.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+module.exports = function (grunt) {
```

Comment:
@misteroneill has been pushing to standardize the rest of the video.js projects around pure npm-scripts for building. While mux.js isn't a traditional video.js plugin, I think it would be good to follow the same conventions unless we have a good reason to deviate.